src/index.js

```
@@ -44,6 +44,7 @@ import { log } from './api/log.js'
 import { merge } from './api/merge.js'
 import { packObjects } from './api/packObjects.js'
 import { pull } from './api/pull.js'
+import { prepareCommit } from './api/prepareCommit.js'
```

ESLint

[error] `./api/prepareCommit.js` import should occur before import of `./api/pull.js`

(import/order)